### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by dominik-ilja** [Dominik-Ilja/solar-conveyance#3]
Requirements :

1. Title: A working title for your app. (Feel free to have some fun with this.)
2. Description: 4 to 5 sentences in non-technical speak summarizing the features, functions, and goals.
3. Wireframes: Mockups of your app on desktop, tablet, and mobile
4. Component Hierarchy: A visual tree of your components, depicting the parent/child relationships, as well as an indication of which will be class components, requiring state, and which will be functional components, taking props.
5. API: The API you will be using and a link to the documentation.
6. MVP: Your goals for MVP, including the minimum, need-to-have features of your app.
7. Post-MVP: Write out what your goals are for post-MVP, including nice-to-have features that you would like to implement once your MVP is complete. (If you plan to use a component library, please link that here as well.)
